larq_zoo/utils.py

```diff
@@ -11,6 +11,27 @@
 ImagenetDataset = namedtuple("ImagenetDataset", ["input_shape", "num_classes"])
 
 
+def slash_join(*args):
+    return "/".join(arg.strip("/") for arg in args)
+
+
+def download_pretrained_model(model, version, file, file_hash, cache_dir=None):
```

Comment:
How do you feel about consolidating `file` and `version` into a single argument called `tag`?

Reply:
doesn't work if we want to leave out the version in the locally stored copy